compatibility/bazel_tools/testing.bzl

```diff
@@ -1139,7 +1139,7 @@ def sdk_platform_test(sdk_version, platform_version):
                     server = ":sandbox-with-postgres-{}".format(platform_version),
                     server_args = [platform_version, "sandbox-on-x", "--participant participant-id=example,port=6865,server-jdbc-url=__jdbcurl__"] + sandbox_on_x_args + extra_sandbox_on_x_args,
                     tags = ["exclusive", sdk_version, platform_version] + extra_tags(sdk_version, platform_version),
-                ) if is_linux else None
+                ) if not is_windows else None
```

Comment:
what’s your goal here? Allowing for local testing or do you really want to run them on CI? If the former, I’d recommend to add a `manual` tag on macos & windows instead of running it. That way, you can select the target explicitly to run it locally but CI won’t run it by default.

Reply:
my intent was to fix some flaky tests as part of the DPP-475 ticket and I did not know how to do it, so I reversed the previous PR mentioned in the ticket: #10178

Did not get about the tag though, you mean git tag or something else?

Reply:
No a bazel tag, we use that in other places already e.g. the oracle migration tests https://github.com/digital-asset/daml/blob/fdb5cdae0a8e9209a8f7fa064e97dd78429c7d1c/compatibility/sandbox-migration/util.bzl#L49